ciri/api/http.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __CIRI_API_API_HTTP_H__
+#define __CIRI_API_API_HTTP_H__
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "cclient/serialization/serializer.h"
+#include "ciri/api/api.h"
+#include "common/errors.h"
+#include "consensus/consensus.h"
+#include "gossip/components/broadcaster.h"
+#include "utils/handles/thread.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct iota_api_http_s {
+  bool running;
+  iota_api_t *api;
```

Comment:
`iota_api_t` also contains a `running` variable and a thread that are now obsolete ?

Reply:
I've removed the thread variable from there - it wasn't being used anyway.